addons/point_of_sale/models/pos_session.py

```diff
@@ -1495,7 +1495,7 @@ def set_cashbox_pos(self, cashbox_value, notes):
         self._post_cash_details_message('Opening', difference, notes)
         #if there is a difference create an account move to register the loss
         if difference:
-            self.env['account.bank.statement.line'].create({
+            self.env['account.bank.statement.line'].sudo().create({
```

Comment:
Rpc method is public so anyone can have access to it. Some restriction need to be put in place like checking if the user belongs to the pos group for example